hera_mc/cm_hookup.py

```diff
@@ -412,11 +483,13 @@ def show_hookup(self, hookup_dict, cols_to_show='all', show_levels=False, show_p
             table = '<html>\n\t<body>\n\t\t<pre>\n' + table + '\t\t</pre>\n\t</body>\n</html>\n'
         print(table, file=file)
 
-    def __make_header_row(self, header_col_list, cols_to_show):
+    def __make_header_row(self, header_col_list, cols_to_show, show_levels):
```

Comment:
I am a bit confused by this approach. Why not just add `level` to `cols_to_show` when one wants to achieve the effect of `show_levels = True`?

Reply:
You can always include "levels" in the list, but if you show-levels you don't necessarily need to or want to remember to do that.  Also, we now have a file to deal with that may or may not have the flag set.  It just sort of made sense with no harm.